Makefile

```diff
@@ -325,6 +326,36 @@ release-qa: require-tag require-gh-token require-s3cmd require-ruby
 		echo "Copying $$VERSIONED" && \
 		$(S3CMD) cp s3://$(S3_BUCKET)/$$NAME s3://$(S3_BUCKET)/$$VERSIONED; \
 	done && \
+	for NAME in $$(ls -1 update_darwin_amd64.bz2 update_linux_386.bz2 update_linux_amd64.bz2 update_windows_386.bz2); do \
+		echo "Copying update binary $$NAME..." && \
+		$(S3CMD) put -P $$NAME s3://$(S3_BUCKET); \
+	done
+
+release-beta: require-s3cmd
+	@BASE_NAME="lantern-installer-qa" && \
+	BETA_BASE_NAME="lantern-installer-beta" && \
+	for URL in $$($(S3CMD) ls s3://$(S3_BUCKET)/ | grep $$BASE_NAME | awk '{print $$4}'); do \
```

Comment:
Instead of looking for files in the current directory (that are probably deleted by now), we list QA files that are already in S3.